generators/chipyard/src/main/scala/Subsystem.scala

```diff
@@ -104,4 +104,29 @@ class SubsystemModuleImp[+L <: Subsystem](_outer: L) extends BaseSubsystemModule
 
   // create file with boom params
   ElaborationArtefacts.add("""core.config""", outer.tiles.map(x => x.module.toString).mkString("\n"))
+
+  // Generate C header with relevant information for Dromajo
+  // THIS IS INCLUDED IN THE `dromajo_params.h` header file
+  var dromajoParams: String = ""
```

Comment:
If it has to be in Subsystem.scala, I would have preferred this to be encapsulated somehow